### ISSUE TITLE
Links to the Russian keyboard layout does not work

### ISSUE THREAD

**Issue by dseter** [philc/vimium#2058]
The latest version of Opera and Yandex Browser on OS X following a link only works when you turn on the English keyboard layout. The Russian keyboard layout reference only indicated by letters, but the transition is not working.

**Comment by smblott-github** [comment]
Could you give more details. @dseter, please?  What are your link-hint settings?  Alphabet hints, yes?  Perhaps a screenshot.  Is this Opera/Yandex only, or Chrome too?

**Comment by dseter** [comment]
Yes, this alphabet hints. Digital hints are working properly. I use them.
Yes, in Google Chrome also not working. And under Windows this problem too.
If the Russian layout, press f, then hints appear, but when you press the key with the required letter, all the hints disappear and nothing happens. I do not know how to show it in the screenshot :)

**Comment by smblott-github** [comment]
What are your link-hint characters (on the options page)?

**Comment by dseter** [comment]
I use Russian language keyboards from this page: https://github.com/philc/vimium/wiki/Key-Mappings.
It is strange, but in Linux hints with Russian keyboard layout works well.

**Comment by fosemberg** [comment]
i have got same problem.
Earlier was every thing well.
When i type "f" to open some link i can not activate link, when i have got Rus keyboard layout.
i need to change keyboard layout to Eng.
Can i type global signs, like
VK_E vk45 E key
VK_F vk46 F key
to work always in both keyboard layout when i want to pick link through "f"
Or maybe to use eatly version of vinium becous in early version it work good (i do not know in wich, but nearly 7 days ago was everything well)
*Rus keys work good in all navigation, but after i type "f" to open link and everything marks with Eng keys i can not use Rus alternative keys For example "y" - "н", "s" - "ы"

ScreenShots:
https://yadi.sk/i/HfJDfqs7sS58p
https://yadi.sk/i/Lugv7_kfsS5CZ
https://yadi.sk/i/U4y1AgvDsS5MQ

**Comment by leoj3n** [comment]
I think this is related to #2147
